sdk/appconfiguration/app-configuration/README.md

```diff
@@ -31,10 +35,8 @@ npm install @azure/ms-rest-nodeauth
 ##### Sample code
 
 ```typescript
-import * as coreHttp from "@azure/core-http";
-import * as coreArm from "@azure/core-arm";
 import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { ConfigurationClient, ConfigurationModels, ConfigurationMappers } from "@azure/app-config";
+import { ConfigurationClient, ConfigurationModels, ConfigurationMappers } from "@azure/app-configuration";
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
 
 msRestNodeAuth.interactiveLogin().then((creds) => {
```

Comment:
Can't click, but we can skip the browser parts since we aren't building the browser version yet.

Reply:
Removed.